client/settings/state/values/actions.js

```diff
@@ -62,7 +63,16 @@ export const submit = ( schema, silent ) => ( dispatch, getState, { callbackURL,
 		}
 	};
 
-	const coercedValues = coerceFormValues( schema, getState().form.values );
+	const coercedValues = coerceFormValues( formSchema, getState().form.values );
```

Comment:
`schema` was already being passed to this function as a parameter, so no need to use the `thunk` property.